source/nodejs/adaptivecards-designer/src/adaptivecards-designer.css

```diff
@@ -1708,3 +1708,13 @@ a.default-ac-anchor:visited:active {
         padding-right: 3px;
     }
 }
+
+@media (max-width: 350px) {
+    input::-webkit-calendar-picker-indicator{
```

Comment:
@jwoo-msft @paulcam206 unsure if we want to remove the icon? Keeping it is slightly misleading since there is no longer a popup, but it is also the only clear identifier that this is a date control.